src/highdicom/content.py

```diff
@@ -859,3 +987,149 @@ def __init__(
             self.IssuerOfTheSpecimenIdentifierSequence.append(
                 issuer_of_specimen_id
             )
+
+
+class LUT(Dataset):
+
+    """Dataset describing a pixel value lookup table."""
+
+    def __init__(
+        self,
+        first_mapped_value: int,
```

Comment:
This appears to be handled via the [Stored Value Color Range Macro](https://dicom.nema.org/medical/dicom/current/output/chtml/part03/sect_C.7.6.16.2.html#sect_C.7.6.16.2.26):

> In case of floating point mapping the second value of the Palette Color Lookup Table Descriptor is not used in the mapping as the Minimum Stored Value Mapped (0028,1231) is used as start value for the LUT values